lib/ansible/modules/file.py

```diff
@@ -552,35 +552,40 @@ def execute_touch(path, follow, timestamps):
     result = {'dest': path}
     mtime = get_timestamp_for_time(timestamps['modification_time'], timestamps['modification_time_format'])
     atime = get_timestamp_for_time(timestamps['access_time'], timestamps['access_time_format'])
+    diff = initial_diff(path, 'touch', prev_state)
+    file_args = module.load_file_common_arguments(module.params)
```

Comment:
These aren't used any sooner, can you put them back?

Reply:
Yes sure!